src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1/package-info.java

```diff
@@ -0,0 +1,5 @@
+/*OK: for test allowing to place annotation before PACKAGE_DEF.*/
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test1.packageinfo;
+
+import java.lang.Deprecated;
```

Comment:
please do import of the same annotations(2 annotations) as in Guava file that I/you mentioned above.